src/common/mod.rs

```diff
@@ -119,14 +119,17 @@ where
         }
     }
 
-    pub fn handshake(&mut self, cx: &mut Context) -> Poll<io::Result<(usize, usize)>> {
+    pub fn handshake(
```

Comment:
Note, this contains a semver-incompatible change.

(I don't remember if there's a good reason to make this public?)

Reply:
The `common` module is not public, so handshake is not actually external